lib/pg_repack.sql.in

```diff
@@ -186,6 +217,8 @@ CREATE VIEW repack.tables AS
          'CREATE TABLE repack.table_' || R.oid || ' WITH (' || array_to_string(array_append(R.reloptions, 'oids=' || CASE WHEN R.relhasoids THEN 'true' ELSE 'false' END), ',') || ') TABLESPACE '  AS create_table_1,
          coalesce(quote_ident(S.spcname), 'pg_default') as tablespace_orig,
          ' AS SELECT ' || repack.get_columns_for_create_as(R.oid) || ' FROM ONLY ' || repack.oid2text(R.oid) AS create_table_2,
+	 'INSERT INTO repack.table_' || R.oid || ' SELECT ' || repack.get_columns_for_create_as(R.oid) || ' FROM ONLY ' || repack.oid2text(R.oid) AS copy_data,
+	 repack.get_alter_col_storage(R.oid) AS alter_col_storage,
```

Comment:
Mix of tabs and space... but I'm afraid this is true a bit everywhere: maybe not worth fixing?